nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -111,16 +112,19 @@ public InferredJARModelsHandler(Config config) {
    * Scan Java Classpath for JarInfer model jars and map their names to locations
    */
   private void processClassPath() {
-    URL[] classLoaderUrls =
-        ((URLClassLoader) Thread.currentThread().getContextClassLoader()).getURLs();
-    for (URL url : classLoaderUrls) {
-      String path = url.getFile();
-      if (path.matches(config.getJarInferRegexStripModelJarName())) {
-        String name = path.replaceAll(config.getJarInferRegexStripModelJarName(), "$1");
-        LOG(DEBUG, "DEBUG", "model jar name: " + name + "\tjar path: " + path);
-        if (!mapModelJarLocations.containsKey(name))
-          mapModelJarLocations.put(name, new LinkedHashSet<>());
-        mapModelJarLocations.get(name).add(path);
+    try (ScanResult scanResult = new ClassGraph().scan()) {
+      for (File file : scanResult.getClasspathFiles()) {
+        String path = file.getAbsolutePath();
+        if (System.getProperty("os.name").startsWith("Windows")) {
+          path = path.replace('\\', '/');
+        }
```

Comment:
This was the simplest change I could figure out to get this code to work on Windows.  On Windows, paths have `\` separators, but URLs still have `/` separators.  And our current regex matching scheme relies on `/` characters and is used in multiple places.  I think overall this code could use some cleanup, but I don't fully understand it, so I went with a narrow fix (will add code comments if it looks ok).